ksonnet-util/util.libsonnet

```diff
@@ -1,7 +1,6 @@
 // util.libsonnet provides a number of useful (opinionated) shortcuts to replace boilerplate code
-local k = import 'grafana.libsonnet';
 
-{
+function(k=(import 'grafana.libsonnet')) {
```

Comment:
this means we need to consume `util.libsonnet` with `local util = import 'ksonnet-util/util.libsonnet'()`, which is unusual, correct?

Reply:
```
local k = import 'grafana.libsonnet';
local _util = import '_util.libsonnet'; // <-- current contents of util.libsonnet go here
_util(k) + {
  withMyK(k): _util(k),
}
```
Now, we can do `import 'ksonnet-util/util.libsonnet'` as normal, but also: `(import 'util.libsonnet').withMyK(this)`

Reply:
Excellent feedback, the top level function would indeed require `(import 'ksonnet-util/util.libsonnet')()`, have a look at 79d4a3c, that should prevent it.